bossdata/remote.py

```diff
@@ -180,7 +180,7 @@ def local_path(self, remote_path):
         """
         if self.local_root is None:
             raise RuntimeError('No local root specified (try setting BOSS_LOCAL_ROOT).')
-        return os.path.abspath(os.path.join(self.local_root, remote_path.lstrip('/')))
+        return os.path.abspath(os.path.join(self.local_root, *remote_path.split('/')))
```

Comment:
I noticed that you removed the `lstrip('/')` that was applied to remote path, which was there to handle cases where `$BOSS_LOCAL_ROOT` ends with a `/` and `$BOSS_SAS_ROOT` begins with a `/` (since both of these are implied whether or not they are explicitly included).  However, it looks like this version handles this just fine.